pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -177,6 +177,19 @@ func (service serviceMetadata) match(rule *rbacproto.AccessRule) bool {
 	return true
 }
 
+// isServiceMatchedWorkload checks if the calling service is matched with the selector in the AuthorizationPolicy if
+// present.
+func (service serviceMetadata) isServiceMatchedWorkload(labels map[string]string) bool {
```

Comment:
Try to use the existing `HasSubsetOf()`: https://github.com/istio/istio/blob/5d16f5623f52b93e7459c9621f8c6e13509379b0/pilot/pkg/model/service.go#L660

Reply:
Done!